libmscore/score.h

```diff
@@ -541,7 +542,10 @@ class Score : public QObject, public ScoreElement {
       void addMeasure(MeasureBase*, MeasureBase*);
       void readStaff(XmlReader&);
       bool read(XmlReader&);
-
+      void setAnnotation()          { _isAnnotation = 1 ; }
```

Comment:
This shouldn't be needed.  Elements already have a type field, this is what the "is" functions test.  See my comment above.